src/Liip/RMT/VCS/VCSInterface.php

```diff
@@ -79,6 +86,14 @@ public function getLocalModifications();
     public function saveWorkingCopy($commitMsg = '');
 
     /**
+     * Revert the last commit. If a message is given, only revert
+     * if the commit message matches.
+     *
+     * @param string|null $commitMsg
+     */
+    public function revertLastCommit($commitMsg = null);
```

Comment:
I don't like that the revertLastCommit is based on the commit message. Since some months I'm thinking about writing more explicit commit message. Currently we have:
<img width="132" alt="zebra__git_" src="https://cloud.githubusercontent.com/assets/114489/11195023/dd02c80c-8caf-11e5-93e6-74dc3f676c78.png">
It's a bit sad, especially if you think that most of the time, we have a message at disposable for the Changelog. So I was think about to write nicer messages. But if we use it in this context, this will block us to do so...